skops/io/_utils.py

```diff
@@ -171,3 +171,21 @@ def get_state(value, save_context):
 
     res["__id__"] = __id__
     return res
+
+
+def get_type_name(
+    t: Any,
+) -> str:
+    """Simple helper function to take in a type, and return its name as a string"""
+    return f"{t.__module__}.{t.__name__}"
+
+
+def cast_defaults_to_strings(defaults: List[Any]) -> List[str]:
```

Comment:
```suggestion
def cast_defaults_to_strings(defaults: List[Any]) -> List[str]:
```

I'm not so happy with the name here. First there is the "cast" part, I'm not sure if this is actually the right word here. Second, I get why you chose "defaults", as that's where this function is used, but it could be used elsewhere and outside of that context, "defaults" doesn't really describe what the function does.

I wonder if we can just name it `get_type_names`, so very similar to the function above. Also, I'm not sure if that function has to be a separate function at all.

Reply:
I separated them so that the logic would be easier to read & test imo, but you're right that it could be a single function as it's currently only used together.

Happy to rework to be a single big function that combines the functionality if you'd prefer

Reply:
> I separated them so that the logic would be easier to read & test imo, but you're right that it could be a single function as it's currently only used together.

Up to you, I don't insist.